.github/workflows/release-script-test.yml

```diff
@@ -0,0 +1,31 @@
+name: Release_Script_Test
+
+on:
+  push:
+    paths:
+      - "scripts/release/**"
+      - ".github/workflows/release-script-test.yml"
+  pull_request:
+    paths:
+      - "scripts/release/**"
+      - ".github/workflows/release-script-test.yml"
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2.4.0
+        with:
+          version: "14"
+          cache: "yarn"
+
+      - name: Install Dependencies
+        run: cd scripts/release && yarn install --frozen-lockfile
```

Comment:
Can we use `working-directory`? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun

Reply:
48fc148 , works fine